README.md

```diff
@@ -295,6 +295,32 @@ In addition to the tags that are specified (e.g., "what" and "endpoint" in this
 | unit        | \<unit\>/s |\<unit\> is what is originally specified as "unit" attribute during declaration. If missing, the value will be set as "n/s". For example if you originally specify .tagged("unit", "request") on a Meter, FfwdReporter emits Meter data points with "unit":"request/s"       |
 | stat | 1m, 5m    | **1m** means the size of the time bucket of the calculated moving average of this data point is 1 minute. **5m** means 5 minutes.         |
 
+
+## Deriving Meter
+It is a regular meter under the hood with slight difference.
```

Comment:
Just thought of a little rewording: 

A deriving meter takes the derivative of a value that is expected to be almost monotonically
increasing.
<BR>A typical use case is to get the rate of change of a counter of the total number of
events.
<BR>This implementation ignores updates that decrease the counter value. The rationale is that the counter is expected to be monotonically increasing between infrequent resets (when a process has been restarted, for example). Thus, negative values should only happen on restart and should be safe to discard.

Reply:
Was debating removing "almost" monotonically as well. Is the almost just referring to restarts?

Reply:
I grabbed that description directly from the code, tbh. I can remove it if it hurts :)

Reply:
removed and updated the desc